### DISCUSSION TITLE
Craft 4 : How to update order email on guest

### DISCUSSION THREAD

**Discussion by aodihis** [craftcms/commerce#2780] [Q&A] [answered]
In craft 3, we can update the cart email address which can be done more than once.
So in craft 4, if we already pass the email for the first time in the cart, can we re-update the email again?
I have done this scenario in Craft 4 but doesn't work.
1. Open shop as a guest.
2. Put an item on the cart.
3. Update the cart with the email address.
4. Repeat step 3.

In step 4 the address is not updated, still use the same email from the step 3.

**Comment by nfourtythree** [comment, marked as answer]
Hi @aodihis 

Thank you for reporting this, I have created an issue for it: https://github.com/craftcms/commerce/issues/2782

We have also pushed an update to the Commerce 4 branch that fixes the issue. This fix will be included in the next Commerce 4 beta release.

Thanks.

**Reply by aodihis** [reply]
Thank you for the answer.
